packages/ui-components/src/icon/labicon.tsx

```diff
@@ -549,10 +549,6 @@ export class LabIcon implements LabIcon.ILabIcon, VirtualElement.IRenderer {
       if (uuid) {
         svgElement.dataset.iconId = uuid;
       }
-
-      if (title) {
-        Private.setTitleSvg(svgElement, title);
-      }
```

Comment:
Why is this being removed? CC @gabalafou

Reply:
Oh, hmm. I think I messed up here. I thought it was safe to remove `setTitleSvg` because it is private and only called on this line and there's only one place in the codebase where `_initSvg` is called and it does not pass `title` (line 375 above):

```
      this._svgElement = this._initSvg({ uuid: this._uuid });
```

But I guess because `_initSvg` is a protected method, it could be called by some other code outside the repo that subclasses `LabIcon`, so I guess the `setTitleSvg` function is not safe to remove, sorry!

Reply:
I have put it back into the file.

Reply:
No sorry I was not clear in my question (I think it is likely fine to remove it, but possibly we should start with a deprecation); the intent/gist of my question is to understand why we do not want to have an option to set a title on the svg icon using this code path because top level comment in this PR does not explain.

Reply:
Ok, sorry that we got back and forth here. The bottom line: if we are removing them, it needs a good rationale and explanation in the top-level comment; otherwise, lets restore it and we can merge this PR as is..